src/Hit/Git/Status.hs

```diff
@@ -136,6 +137,34 @@ parseDiffStat = \case
         }
     _ -> Nothing
 
+{- | Attempts to expand shortened paths which can appear in `git diff --stat`.
+Examples of possible paths:
```

Comment:
Can you, please, add a few examples on how this function works on different expected inputs? IIUC, documentations only shows expected intputs at the moment.